Tools/wasm/README.md

```diff
@@ -54,18 +72,63 @@ CONFIG_SITE=../../Tools/wasm/config.site-wasm32-emscripten \
     --with-build-python=$(pwd)/../build/python
 
 emmake make -j$(nproc)
+popd
 ```
 
-### Test in browser
-
-Serve `python.html` with a local webserver and open the file in a browser.
-
-```shell
-emrun python.html
 ```
-
-or
-
-```shell
-python3 -m http.server
+node --experimental-wasm-threads --experimental-wasm-bulk-memory builddir/emscripten-node/python.js
 ```
+
+## wasm32-emscripten limitations and issues
+
+- Most stdlib modules with dependency on external libraries are missing:
+  ``ctypes``, ``readline``, ``sqlite3``, ``ssl``, and more.
+- Shared extension modules are not implemented yet. All extension modules
+  are statically linked into the main binary.
+- Processes are not supported. Calls like fork, popen, and subprocess
+  fail with ``ENOSYS`` or ``ENOSUP``.
+- Blocking sockets are not available. Non-blocking sockets don't work
+  correctly, e.g. ``socket.accept`` crashes the runtime. ``gethostbyname``
+  does not resolve to a real IP address. IPv6 is not available.
+- The ``select`` module is limited. ``select.select()`` crashes the runtime
+  due to lack of exectfd support.
+- The ``*at`` variants of functions (e.g. ``openat``) are not available.
+  The ``dir_fd`` argument of *os* module functions can't be used.
+- Signal support is limited. ``signal.alarm``, ``itimer``, ``sigaction``
+  are not available or do not work correctly. ``SIGTERM`` exits the runtime.
+- Most user, group, and permission related function and modules are not
+  supported or don't work as expected, e.g.``pwd`` module, ``grp`` module,
+  ``os.setgroups``, ``os.chown``, and so on.
+- Offset and iovec I/O functions (e.g. ``os.pread``, ``os.preadv``) are not
+  available.
+- ``os.mknod`` and ``os.mkfifo`` don't work and are disabled.
+- Large file support crashes the runtime and is disabled.
+- ``mmap`` module is unstable. flush (``msync``) can crash the runtime.
+- Resource-related functions like ``os.nice`` and most functions of the
+  ``resource`` module are not available.
+- Some time and datetime features are broken. ``strftime`` and ``strptime``
+  have known bugs. Extended glibc formatting features are not available.
+- ``locales`` module is affected by musl libc issues.
```

Comment:
Link to the bug?

Reply:
I have added links to bpo and emscripten bug tracker.